### ISSUE TITLE
Sourcemaps are not working properly

### ISSUE THREAD

**Issue by juliomatcom** [drcallaway/apollo-link-timeout#62]
Hi,
I'm having the next problem with the sourcemap when compiling my project:

> Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '[dir]/node_modules/apollo-link-timeout/src/TimeoutError.ts' file: Error: ENOENT: no such file or directory, open '[dir]/node_modules/apollo-link-timeout/src/TimeoutError.ts'

this is because in the package.json `files` config is set to only publish the `lib` folder https://github.com/drcallaway/apollo-link-timeout/blob/master/package.json#L46

Then what happens is that the sourcemap generated, e.g. for `TimeoutError.js` will point to the original file in `src` which is not included in the package.

> {"version":3,"file":"TimeoutError.js","sourceRoot":"","sources":["../src/TimeoutError.ts"],"names":[],"mappings":";;;;;;;;;;;;AAAA;IAA0C,gCAAK;IAI7C,sBAAY,OAAe,EAAE,OAAe,EAAE,UAAwB;QAAxB,2BAAA,EAAA,gBAAwB;QAAtE,YACE,kBAAM,OAAO,CAAC,SAGf;QAFC,KAAI,CAAC,OAAO,GAAG,OAAO,CAAC;QACvB,KAAI,CAAC,UAAU,GAAG,UAAU,CAAC;;IAC/B,CAAC;IACH,mBAAC;AAAD,CAAC,AATD,CAA0C,KAAK,GAS9C"}

A fix could be just to add the src folder to the distributable in the `package.json` and publish a new version
```
  "files": [
    "lib",
    "src"
  ],
```
Let me know if I can do it myself and open PR, thank you!

**Comment by davidpenso** [comment]
any luck on solving this @juliomatcom ?

**Comment by juliomatcom** [comment]
> any luck on solving this @juliomatcom ?

It looks like this package is unmaintained, I suggest you make your own fork/ copy and fix it there as I explained above. Use your own version for now.

**Comment by drcallaway** [comment]
That is correct. This package is no longer maintained but I’d be happy to transfer it to anyone that is interested in doing that.
